pilot/pkg/model/push_context.go

```diff
@@ -360,6 +360,20 @@ type PushRequest struct {
 	// There should only be multiple reasons if the push request is the result of two distinct triggers, rather than
 	// classifying a single trigger as having multiple reasons.
 	Reason []TriggerReason
+
+	Delta ResourceDelta
```

Comment:
Is this the current pushrequest's? Worth a comment

Reply:
Yes, let me add a comment